test/wdio/external-imports/cmp.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { Fragment, h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('external-imports', () => {
+  beforeEach(async () => {
+    render({
+      template: () => (
+        <>
+          <external-import-a></external-import-a>
+          <external-import-b></external-import-b>
+          <external-import-c></external-import-c>
+        </>
+      )
+    })
+  });
+  it('render all components without errors', async () => {
+    let elm = $('external-import-a');
+    await expect(elm).toHaveText('Marty McFly');
+
+    elm = $('external-import-b');
+    await expect(elm).toHaveText('Marty McFly');
```

Comment:
I know this was how it was in the original test, but can we use a separate variable assignment here in the off-chance that this assignment doesn't resolve/has a reference to `external-import-a` (unsure how possible that actually is in practice)? I want to make sure we're asserting against `external-import-b` instead of `external-import-a`:
```suggestion
    const elm2 = $('external-import-b');
    await expect(elm2).toHaveText('Marty McFly');
```